.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,30 @@
+# Doc: https://github.com/redhat-plumbers-in-action/differential-shellcheck#usage
+---
+
+name: Differential ShellCheck
+on:
+  pull_request:
+    branches: [master]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    permissions:
+      security-events: write
```

Comment:
Why is this added? It seems unnecessary, this is not a private repository

Reply:
I usually use `permissions` to set the minimum required permissions for the job. And to be honest, I'm not sure what will happen when you leave out `security-events: write`, I don't know what the default permissions are for the forks. I'm going to test that.

Reply:
Yes, you are right. It doesn't have to be set.
- https://github.com/jamacku/ruby-build/pull/1

Would you prefer the removal of `permissions`?

Reply:
Let's remove `security-events: write`, but keep `pull-requests: write`

Reply:
Removed.